app/Http/Controllers/StatusPageController.php

```diff
@@ -109,7 +109,7 @@ public function showIndex()
             ->withDaysToShow($daysToShow)
             ->withAllIncidents($allIncidents)
             ->withCanPageForward((bool) $today->gt($startDate))
-            ->withCanPageBackward(Incident::notScheduled()->where('created_at', '<', $startDate->format('Y-m-d'))->count() > 0)
+            ->withCanPageBackward(Incident::notScheduled()->where('occurred_at', '<', $startDate->format('Y-m-d'))->count() > 0)
```

Comment:
Yes, because `occurred_at` might contain a time, but we're only interested in whole days.